storybook/stories/API/chart/RadialBarChart.stories.tsx

```diff
@@ -1,30 +1,55 @@
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 /* eslint-disable jsx-a11y/click-events-have-key-events */
+import { Meta } from '@storybook/react';
 import React, { useState } from 'react';
 import { pageData, pageDataWithFillColor } from '../../data';
-import { RadialBar, RadialBarChart, ResponsiveContainer, Cell, Legend } from '../../../../src';
+import { Tooltip, RadialBar, RadialBarChart, ResponsiveContainer, Cell, Legend, RadialBarProps } from '../../../../src';
 import { CategoricalChartProps } from '../props/ChartProps';
 
 export default {
   argTypes: {
     ...CategoricalChartProps,
+    activeIndex: {
+      description:
+        'The index of the individual shapes that make up the Funnel to be marked as active, and render props.activeShape as a result',
+      table: {
+        type: {
+          summary: 'number | undefined',
+        },
+        defaultValue: undefined,
```

Comment:
Do we need to define these here? Do they not get auto-filled from the code? 🤔

Reply:
Should be fine. I removed these.